.github/PULL_REQUEST_TEMPLATE.md

```
@@ -1,13 +1,19 @@
-- Related GitHub Issue: #569
-
-## Changes
-- [ ] What changed
-- [ ] Why it changed
-- [ ] How to test
-
-## Checklist
-- [ ] Tests added/updated
-- [ ] Documentation updated
-- [ ] Jira issue status updated
+## 🤖 AI-SRE Context
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)